checker/tests/nullness/Issue1142.java

```diff
@@ -0,0 +1,14 @@
+// Issue 1142 https://github.com/typetools/checker-framework/issues/1142
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue1142 {
+
+    void foo() {
+        //:: error: (type.argument.type.incompatible)
+        ConcurrentHashMap<Integer, @Nullable Integer> chm1 = new ConcurrentHashMap<>();
+
+        //:: error: (argument.type.incompatible)
+        chm1.put(1, null);
```

Comment:
Why is this one failing? Won't it use `@Nullable Integer` as the value type and allow this?